backend/app/models/user.py

```diff
@@ -19,6 +20,7 @@ class User(db.Model):
         nullable=False,
     )
     email = db.Column(db.String, nullable=False)
+    last_modified = db.Column(db.DateTime, default=func.now(), onupdate=func.now())
```

Comment:
So I think the issue with what you were dealing with was that by default when a User was queried, the `last_modified` column was returned. In order to handle this, you had to update the DTOs for it to work.

I did some research and we can actually get around this by using `defer`. What `defer` does is it doesn't return a column from a query by default unless explicitly stated. So, if we set `defer=True` on `last_modified`, we should never get it back from a query to the Users table. We should also set it as non-nullable. So, the model will look something like this:

```suggestion
    last_modified = db.Column(db.DateTime, default=func.now(), onupdate=func.now(), defer=True, nullable=False)
```

Once you do this, you should be able to remove all of the updates to `auth_routes` and `resources` that you made